corehq/apps/reports/standard/cases/filters.py

```diff
@@ -131,16 +139,23 @@ def get_column_suggestions(self):
         special_properties = [
             {'name': prop, 'case_type': None, 'meta_type': 'info'}
             for prop in SPECIAL_CASE_PROPERTIES + CASE_COMPUTED_METADATA
+            if prop not in self.exclude_special_case_properties
         ]
         return case_properties + special_properties
 
+    @property
+    def exclude_special_case_properties(self):
```

Comment:
Instead of copying code in `CaseListExplorerColumns` and `CaseSearchFilter`, you could drop this method from both classes (DRY), and replace the `special_properties` or `special_case_properties` variables with a function call to something like this:
```python
def get_special_case_properties(domain):
    from corehq.apps.reports.filters.api import CaseCopier

    props = SPECIAL_CASE_PROPERTIES + CASE_COMPUTED_METADATA
    if not toggles.COPY_CASES.enabled(domain):
        props -= [CaseCopier.COMMCARE_CASE_COPY_PROPERTY_NAME]
    return [
        {'name': prop, 'case_type': None, 'meta_type': 'info'}
        for prop in props
    ]
```